Core/Controller/EditPageOption.php

```diff
@@ -134,6 +140,34 @@ private function saveData()
         $this->miniLog->alert($this->i18n->trans('data-save-error'));
     }
 
+    /**
+     * Delete configuration for view
+     */
+    private function deleteData()
+    {
+        $nick = $this->request->get('nick');
+        if (!$nick) {
+            $where = [
+                new Base\DataBase\DataBaseWhere('nick', 'null', 'IS'),
+                new Base\DataBase\DataBaseWhere('name', $this->selectedViewName)
+            ];
+        } else {
+            $where = [
```

Comment:
In this case, you don't put this condition, but keep it in mind to avoid else use when it's possible.

This assignment can be before if, and the else statement is not required in this case.

Codacy always warn this cases with this message:

> The method saveData uses an else expression. Else is never necessary and you can simplify the code to work without else.

In cases like this the problem is "is better assign and re-assing" or "use better statement", both cases are good. @shakaran what is your opinion about this?